src/test/java/kitchenpos/application/menu/MenuServiceTest.java

```diff
@@ -0,0 +1,219 @@
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+
+class MenuServiceTest extends ApplicationTestConfig {
+
+    private MenuService menuService;
+
+    @BeforeEach
+    void setUp() {
+        menuService = new MenuService(
+                menuDao,
+                menuGroupDao,
+                menuProductDao,
+                productDao
+        );
+    }
+
+    @DisplayName("메뉴 가격")
+    @Nested
+    class MenuPriceNestedTest {
+
+        @DisplayName("[SUCCESS] 등록할 신규 메뉴에 메뉴 상품이 없을 경우 메뉴의 가격은 0원이어야 등록할 수 있다.")
+        @Test
+        void success_create_menu_when_MenuProductsIsEmpty() {
+            // given
+            final MenuGroup savedMenuGroup = menuGroupDao.save(new MenuGroup("테스트용 메뉴 그룹명"));
+
+            final BigDecimal zeroPrice = BigDecimal.ZERO;
+            final Menu expected = new Menu(
+                    "테스트용 메뉴명",
+                    zeroPrice,
+                    savedMenuGroup.getId(),
+                    Collections.emptyList()
+            );
+
+            // when
+            final Menu actual = menuService.create(expected);
+
+            // then
+            assertSoftly(softly -> {
+                softly.assertThat(actual.getId()).isPositive();
+                softly.assertThat(actual.getName()).isEqualTo(expected.getName());
+                softly.assertThat(actual.getPrice()).isEqualByComparingTo(expected.getPrice());
+                softly.assertThat(actual.getMenuGroupId()).isEqualTo(expected.getMenuGroupId());
+                softly.assertThat(actual.getMenuProducts()).isEqualTo(expected.getMenuProducts());
+            });
+        }
+
+        @DisplayName("[EXCEPTION] 등록할 신규 메뉴에 가격이 null 일 경우 예외가 발생한다.")
+        @Test
+        void throwException_when_create_Menu_IfMenuPriceIsNull() {
+            // given
+            final MenuGroup savedMenuGroup = menuGroupDao.save(new MenuGroup("테스트용 메뉴 그룹명"));
+
+            // when
+            final Menu expected = new Menu(
+                    "테스트용 메뉴명",
+                    null,
+                    savedMenuGroup.getId(),
+                    Collections.emptyList()
+            );
+
+            // then
+            assertThatThrownBy(() -> menuService.create(expected))
+                    .isInstanceOf(IllegalArgumentException.class);
+        }
+
+        @DisplayName("[EXCEPTION] 등록할 신규 메뉴에 가격이 음수일 경우 예외가 발생한다.")
+        @ParameterizedTest
+        @ValueSource(strings = {"-1", "-10", "-100", "-1000000"})
```

Comment:
```suggestion
        @ValueSource(ints = {-1, -10, -100, -1000000})
```
ints 를 사용해보는 건 어떨까요??😊

Reply:
BigDecimal의 경우 소수점이라면 문자열을 넣어 생성하는게 정확한 것으로 알고 있습니다.
물론 현재 제가 적은 값들은 모두 정수여서 상관이 없지만 기존에 BigDecimal을 이용하고 있다는 점에서
이후에 소수점까지 고려하게 될 수도 있다는 것을 고려헀습니다.
때문에 BigDecimal을 보다 정확한 값으로 사용하기 위해서 문자열을 이용했습니다.

Reply:
그렇군요!! 좋은 지식 알아갑니다!! 감사합니다👍